index.html

```diff
@@ -247,16 +250,26 @@ <h5 class="subtitle">LET'S CONNECT</h5>
         </ul>
       </div>
     </section>
-    <section class="container contact-container">
+    <section class="contact-container">
       <div class="contact left-right-margin">
         <h1>Contact Me</h1>
         <p>
           If you have an application you are interested in developing, a feature
           that you need built or a project that needs coding. I’d love to help
           with it
         </p>
-        <form class="contact-form" action="https://formspree.io/f/mqkoqbre" method="post">
-          <input name="name" type="text" placeholder="name" maxlength="30" required />
+        <form
+          class="contact-form"
+          action="https://formspree.io/f/mqkoqbre"
+          method="post"
+        >
+          <input
+            name="name"
+            type="text"
+            placeholder="name"
+            maxlength="30"
+            required
+          />
```

Comment:
- [ ] I would suggest you please use the background image on your contact form. Please see the image below for reference.
- [ ] I would suggest you please remove the border radius to match the design and also increase the height of the input field for text.
### Your design
![his](https://user-images.githubusercontent.com/81727988/220853067-c0df643a-e7c2-43a7-baca-91e16c333d8f.png)

### Figma design

![fimga](https://user-images.githubusercontent.com/81727988/220853306-3060bfa9-2c27-4a99-b082-8b06c40abbf0.png)